.env.example

```diff
@@ -3,6 +3,9 @@ LOGGING_LEVEL=info
 
 WEB_CONCURRENCY=1
 
+AWS_ACCESS_KEY_ID=keygoeshere
+AWS_SECRET_ACCESS_KEY=keygoeshere
```

Comment:
Hm - Actually I think that check [would still run the profiler,](https://github.com/DoSomething/gambit/blob/8dc132d0e0999f9cb4d85a10d618cf2d33f631eb/server.js#L69) as the config variable would be set to a string. Just seems like we'd want to add `V8_PROFILER_ENABLED` if we're adding the `AWS` variables.